conf/modules.config

```
@@ -1700,18 +1615,18 @@ process {
     }
 
     withName: GATK4_HAPLOTYPECALLER {
-        tag = { "${meta.reference}|${meta.sample_id}" }
-        ext.args = {[
-            // Option names have changed from underscore_separated to hyphen-separated in GATK4
-            "--sample-ploidy ${meta2.ploidy}",
-            "-stand-call-conf ${params.genotyping_gatk_call_conf}",
-            "--output-mode ${params.genotyping_gatk_hc_out_mode}",
-            "--emit-ref-confidence ${params.genotyping_gatk_hc_emitrefconf}",
-        ].join(' ').trim() }
+        tag        = { "${meta.reference}|${meta.sample_id}" }
+        ext.args   = {
+            [
+                "--sample-ploidy ${meta2.ploidy}",
```

this should be changed to behave the same as for FREEBAYES, since if the user is utilizing a multiref input, the ploidy can be left as an empty field in the meta.  (credit @jfy133 )

```suggestion
                "--sample-ploidy ${meta2.ploidy}",
                "            ref_meta.ploidy ? "--sample-ploidy ${ref_meta.ploidy}" : "--sample-ploidy 2", 

```